### ISSUE TITLE
Touch

### ISSUE THREAD

**Issue by divarvian** [springmusk026/Imgui-Unity#2]
on some games Like Mobile Legends cant touch (freeze), can you change the touch support mobile legends?

**Comment by springmusk026** [comment]
yes i have been already informed about it but currently i am busy so i am not able to see i will fix it soon with help of my friends

**Comment by ToshiroZ** [comment]
Sorry but he can't fix it as he didn't actually write any of the code
